tests/integration/high_availability/manifests/extend_pebble_restart_delay.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  mysqld:
+    override: merge
+    backoff-delay: 300s
+    backoff-limit: 300s
```

Comment:
If you don't want to rely on/wait for the back off, you can set `on-success` and `on-failure` directives to `ignore`. That way pebble wouldn't restart the services until you explicitly replan. I did something similar [here](https://github.com/canonical/mongodb-k8s-operator/blob/ha-tests-full-cluster-crash/tests/integration/ha_tests/test_ha.py#L86).

Reply:
The extra manifest is no longer needed.

Reply:
Addressed in [d19898e](https://github.com/canonical/mysql-k8s-operator/pull/123/commits/d19898ec30ad24dac69a22318488726a73c3d5d9) and [a40d743](https://github.com/canonical/mysql-k8s-operator/pull/123/commits/a40d74300eec78d52ddcb4d8004870db3070c214)